sos/plugins/nodejs.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2016 Red Hat, Inc., Tomas Tomecek <ttomecek@redhat.com>
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin, \
+    SuSEPlugin
+
+
+class NodeJS(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin, SuSEPlugin):
+    """ Get runtime version of NodeJS """
+
+    plugin_name = 'nodejs'
+    profiles = ('system',)
+
+    packages = ('nodejs',)
```

Comment:
Ubuntu you need to install the `nodejs-legacy` package to get the `node` symlink. Otherwise because of a package already claiming the /usr/bin/node binary the existing `nodejs` package uses `/usr/bin/nodejs` as its binary.